.travis.yml

```diff
@@ -1,4 +1,6 @@
-language: clojure
-script: lein test-all
+language: generic
+services:
+  - docker
+script: sh -c 'docker run -i --rm -v $(pwd):/work -w /work clojure:openjdk-8-tools-deps-alpine clojure -A:test:runner'
```

Comment:
Not that we have great tests in this project, but the previous version ran tests against all supported versions of clojure (`1.{7,8,9,10}`)

Reply:
@tanzoniteblack do you have an example for doing this with `deps.edn`?

Reply:
I do not. A quick google search doesn't find anything, so I think we'll have to write a bash loop or something similar?